source/fundamentals/transactions.txt

```diff
@@ -0,0 +1,154 @@
+.. _golang-transactions:
+
+============
+Transactions
+============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use **transactions** with the
+{+driver-long+}. :manual:`Transactions </core/transactions/>` allow you
+to run a series of operations that are not visible from outside the
+transaction until it is committed. If any operation in the
+transaction fails, the driver aborts the transaction and discards all
+data changes before they ever become visible.
+
+In MongoDB, transactions run within logical **sessions**. A
+:manual:`session </reference/server-sessions/>` is a
+grouping of related read or write operations that you want to run
+sequentially. Sessions enable :manual:`causal consistency </core/read-isolation-consistency-recency/#causal-consistency>`
+for a group of operations or allow you to execute operations in an :website:`ACID
+transaction </basics/acid-transactions>`. MongoDB guarantees that the data
+involved in your transaction operations remains consistent, even if the operations
+encounter unexpected errors.
+
+With the {+driver-short+}, you can create a new session from a
+``Client`` instance as a ``Session`` type.
+
+.. warning::
+
+   You should only use ``Session`` with the ``Client`` (or associated
+   ``Database`` or ``Collection``) that created the ``Session``. Using a
```

Comment:
Issue: "only" should modify "the ``Client``", not "use". [Style Guide link. ](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#o)

Suggestion: 
```suggestion
   You should use a ``Session`` only with the ``Client`` (or associated
   ``Database`` or ``Collection``) that created it. Using a
```